python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -129,6 +129,9 @@ def load(self) -> bool:
             model_id_or_path, do_lower_case=self.do_lower_case, device_map=self.device_map)
         if not self.tokenizer.pad_token:
             self.tokenizer.add_special_tokens({'pad_token': '[PAD]'})
+        if self.tokenizer.pad_token is None:
```

Comment:
Do you mean we need to add the special token for both cases? then we do not need the conditions

Reply:
Sorry, my rebase got messed up. will remove the extra condition